Sources/Combine/Extensions/NotificationCenter/NotificationCenter+Publishers.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NotificationCenter+Publishers.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 15.09.2022..
+//  Copyright © 2022 Infinum. All rights reserved.
+//
+
+import Foundation
+import Combine
+import UIKit
+
+@available(iOS 13.0, *)
+extension NotificationCenter {
```

Comment:
I'm sorry, I don't really get the point of this file 😅

Reply:
What's confusing you? 😄 It's just a small convenience over the `NotificationCenter` so that we don't have to write same 3 old lines. Not necessarily the most useful thing we have, but the call site is a bit prettier.